tests/c2pa.py

```
@@ -1550,43 +1566,42 @@ def sign(self, signer: Signer, format: str, source: Any, dest: Any = None) -> Op
 
     def sign_file(self, source_path: Union[str, Path], dest_path: Union[str, Path], signer: Signer) -> tuple[int, Optional[bytes]]:
         """Sign a file and write the signed data to an output file.
-        
+
         Args:
             source_path: Path to the source file
             dest_path: Path to write the signed file to
-            signer: The signer to use
-            
+
         Returns:
             A tuple of (size of C2PA data, optional manifest bytes)
-            
+
         Raises:
             C2paError: If there was an error during signing
         """
         if not self._builder:
             raise C2paError(self._error_messages['closed_error'])
-            
+
         source_path_str = str(source_path).encode('utf-8')
         dest_path_str = str(dest_path).encode('utf-8')
         manifest_bytes_ptr = ctypes.POINTER(ctypes.c_ubyte)()
-        
+
         result = _lib.c2pa_builder_sign_file(
             self._builder,
             source_path_str,
             dest_path_str,
             signer._signer,
             ctypes.byref(manifest_bytes_ptr)
         )
-        
+
         if result < 0:
             _handle_string_result(_lib.c2pa_error())
```

Here, we can also silently fail